### PR TITLE
[ARCH.2.1b] Align architecture policy with the audited refactor tranche

### LINKED ISSUE #154: [ARCH.2.1b] Align architecture policy with the audited refactor tranche
Parent: #133. Depends on #142 and must merge before #134.

## Defect

The first guardrail baseline predates the independent plan re-audit. Its human roadmap skips the actual trainer prerequisite chain, and several machine-readable dependency exceptions point to issues that cannot remove the named Cargo edge:

- #140 cannot remove `wow-world -> wow-handler` or `wow-world -> wow-packet` by moving impl blocks inside the same crate;
- #137 cannot remove every `wow-world -> wow-network` use because session manager/link/write-fence transport contracts remain;
- registry relocation and registry encapsulation are now intentionally separate slices.

Leaving those assignments unchanged would violate the policy rule that a completed issue must not remain the supposed owner of unresolved debt.

## Goal

Synchronize the architecture document and executable debt ledger with the audited, achievable first-tranche sequence before implementation resumes.

## Scope

- Update `docs/architecture/ownership-and-boundaries.md` with the real prerequisite/order chain #148 -> #144 -> #142 -> #154 -> #134 and the corrected first-tranche children #150-#153.
- Update hotspot evidence and distinguish first-tranche completion from epic-level completion.
- Retarget each misassigned dependency exception to the concrete final open slice that can remove it, or to #153 when the correct long-term boundary requires the mandated post-tranche decision.
- Keep generic #133 exceptions only where the parent genuinely remains open and #153 explicitly owns their re-audit.
- Add self-test fixtures that reject malformed/duplicate debt ownership and an exception whose tracking issue is absent from the checked-in architecture issue ledger; do not make CI depend on live GitHub availability.
- Do not change Cargo dependencies, runtime code, handlers, packets or gameplay behavior.

## Acceptance

- Human sequence, checked-in issue ledger and JSON policy agree.
- No exception claims that #137/#140 will remove an edge their scopes cannot remove.
- The checker remains deterministic/offline and all architecture self-tests pass.
- Full preflight, CI and current-HEAD Codex verdict are green.

Refs #133, #135, #137, #140, #150, #151, #152 and #153.

### PR DESCRIPTION
**Pull request by alseif0x**
## Summary

Closes #154.

Synchronizes the architecture baseline with the audited first-tranche sequence before implementation resumes:

- **New checked-in issue ledger** `tools/architecture/architecture-issue-ledger.json`: every architecture issue with its open/closed state plus the real prerequisite chain (#148 → #144 → … → #142 → #154 → #134, then the corrected first-tranche children #150-#153). Validation is deterministic and offline; CI never calls GitHub.
- **Retargeted dependency exceptions**: `wow-world -> wow-handler` and `wow-world -> wow-packet` move from #140 to #153 (#152/#140 only extract private modules inside `wow-world` and cannot remove those Cargo edges); `wow-world -> wow-network` moves from #137 to #153 (the gameplay uses retire across #134/#138/#150/#137/#151 in order, but the session manager, session-link and write-fence transport contracts remain). No exception now claims #137/#140 will remove an edge their scopes cannot remove.
- **Executable debt-ownership guardrails**: the checker rejects an exception whose `tracking_issue` is absent from the ledger or owned by a completed issue, validates the ledger shape, and fails when the human refactor sequence in `ownership-and-boundaries.md` disagrees with the ledger or the JSON policy.
- **Six new self-test fixtures** under `fixtures/debt-ownership/` prove the rejections: malformed/duplicate debt ownership, tracking absent from the ledger, ownership by a completed issue, malformed/duplicate ledger entries.
- **Doc update**: corrected 18-step refactor sequence, ledger contract, first-tranche hotspot evidence measured at HEAD `c697827c` (SessionResources 228 fields, `create_session` ~812 lines, `session.rs` 156,394 lines, `misc.rs` 198 registrations / 15 Calendar), and the explicit distinction between first-tranche completion and epic-level completion.

No Cargo dependency, runtime code, handler, packet, or gameplay behavior changes.

## Validation

- `python3 tools/architecture/check_architecture.py self-test` → PASS (20 fixtures, 6 debt-ownership rejections)
- `python3 tools/architecture/check_architecture.py check` → PASS (34 packages, 108 workspace edges, 21 workspace + 3 external baseline exceptions)
- `git diff --check` clean; `./tools/pr-preflight.sh quick origin/3.4.3` PASS
- `./tools/pr-preflight.sh full origin/3.4.3` PASS (exit 0; capture-diff required flows CLEAN; local Codex review CLEAN, 0 findings)

### CODE REVIEW

**Comment by alseif0x** [comment]
@codex review

**Comment by alseif0x** [comment]
@codex review

File: tools/architecture/architecture-issue-ledger.json [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+      "title": "[ARCH.2.7a] Extract WorldSession packet admission and dispatch"
+    },
+    {
+      "number": 153,
+      "state": "open",
+      "title": "[ARCH.2.8] Re-audit architecture after the first modularization tranche"
+    },
+    {
+      "number": 154,
+      "state": "open",
```
[the remark this answers is not in the document]
Comment (alseif0x) [reply tools/architecture/architecture-issue-ledger.json]: Fixed in `45dcae05`: the ledger now records #154 as closed, so the offline source of truth is already correct when this PR merges and no future exception can name #154 as an open debt owner without failing the check.

[the remark this answers is not in the document]
Comment (alseif0x) [reply tools/architecture/architecture-issue-ledger.json]: Fixed in `45dcae05` (duplicate thread of the same finding): #154 is marked closed in the ledger within this closing change.

File: tools/architecture/architecture-issue-ledger.json [hunk 2]
```diff
@@ -0,0 +1,146 @@
+{
+  "schema_version": 1,
+  "parent_issue": 133,
+  "reaudit_issue": 153,
+  "sequence": [
+    135,
+    148,
+    144,
+    163,
```
[the remark this answers is not in the document]
Comment (alseif0x) [reply tools/architecture/architecture-issue-ledger.json:12]: Fixed in `45dcae05`: the ledger and the documented sequence now include the omitted closed prerequisites #143 (C++ interaction provenance), #146 (effective SpellInfo key authority) and #156 (independent primary-profession capacity), restoring the full audited chain #135 -> #143 -> #146 -> #148 -> #144 -> #156 -> #163 -> #164 -> #157-#161 -> #142 -> #154 -> #134 -> ... -> #153.

File: docs/architecture/ownership-and-boundaries.md [hunk 3]
```diff
@@ -223,21 +230,49 @@ Do not regenerate a baseline merely to make CI green.
    concrete removal issue.
 6. Run the architecture self-test, architecture check, focused Rust tests, and full PR preflight.
 
+## First-tranche hotspot evidence
+
+Measured at the #154 baseline (HEAD `c697827c`); refresh these numbers as tranche PRs land:
+
+- `wow-network::accept::SessionResources` carries 228 public fields in
```
[the remark this answers is not in the document]
Comment (alseif0x) [reply docs/architecture/ownership-and-boundaries.md]: Fixed in `45dcae05`: re-measured at the stated baseline — `SessionResources` spans `crates/wow-network/src/accept.rs` lines 190-513 with 244 `pub` field declarations; the hotspot evidence now records 244 and the exact line span.
